examples/scripts/Custom NS with VW learner.py

```diff
@@ -0,0 +1,57 @@
+"""
+This is an example script that creates and execuates an Experiment.
+This script requires that the matplotlib and vowpalwabbit packages be installed.
+"""
+
+import coba as cb
+from coba.learners import VowpalMediator
+from typing import List
+
+# this mediator splits the 'x' namespace into multiple namespaces
+class CustomVWMediator(VowpalMediator):
```

Comment:
could also be an environment filter but it has been useful for some VW specific engineering

Reply:
Yeah, I've thought about adding an environment filter for this purpose for a while... VW isn't the only learner in coba that uses namespaces. That being said, it'd be a little tricky which is why I haven't done it yet. So, I think this is great for now. 

Also, nice thinking to create a custom mediator. It's a great solution.